build.cmd

```diff
@@ -307,6 +307,10 @@ if defined __SkipConfigure goto SkipConfigure
 
 echo %__MsgPrefix%Regenerating the Visual Studio solution
 
+:: Generate _version.h
```

Comment:
Please move this further up so that this header is generated independent of whether we skip native or managed build. The place to do this would be right after the comment above that says "Start the build steps".